hoomd/md/minimize/fire.py

```diff
@@ -0,0 +1,206 @@
+
+    At each time step, :math:`\\delta t`, the algorithm uses the NVE Integrator
+    to generate a x, v, and F, and then adjusts v according to
+
+    .. math::
+
+        \\vec{v} = (1-\\alpha)\\vec{v} + \\alpha \\hat{F}|\\vec{v}|
+
+    where :math:`\\alpha` and :math:`\\delta t` are dynamically adaptive
+    quantities.  While a current search has been lowering the energy of system
+    for more than :math:`N_{min}` steps, :math:`\\alpha` is decreased by
+    :math:`\\alpha \\rightarrow \\alpha fdec_{\\alpha}` and :math:`\\delta t`
+    is increased by :math:`\\delta t \\rightarrow max(\\delta t \\cdot
+    finc_{dt}, \\ \\delta t_{max})`. If the energy of the system increases (or
+    stays the same), the velocity of the particles is set to 0, :math:`\\alpha
+    \\rightarrow \\ \\alpha_{start}` and :math:`\\delta t \\rightarrow \\delta t
+    \\cdot fdec_{\\alpha}`. Convergence is determined by both the force per
+    particle and the change in energy per particle dropping below `angmom_tol`
+    and `energy_tol`, respectively or
+
+    .. math::
+
+        \\frac{\\sum |F|}{N*\\sqrt{N_{dof}}} < \\mathrm{\\text{angmom_tol}}
+        \\;\\; and \\;\\ \\Delta \\frac{\\sum|E|}{N} <
+        \\mathrm{\\text{energy_tol}}
+
+    where N is the number of particles the minimization is acting over (i.e.
+    the group size) Either of the two criterion can be effectively turned off
+    by setting the tolerance to a large number.
+
+    If the minimization is acted over a subset of all the particles in the
+    system, the "other" particles will be kept frozen but will still interact
+    with the particles being moved.
+
+    Examples::
+
+        fire = md.minimize.FIRE(dt=0.05)
+        fire.force_tol = 1e-2
+        fire.energy_tol = 1e-7
+        fire.methods.append(methods.NVE(filter.All()))
+        sim.operations.integrator = fire
+        while not(fire.converged):
+           sim.run(100)
+
+    Examples::
+
+        fire = md.minimize.FIRE(dt=0.05)
+        fire.methods.append(methods.NPH(filter.All(), S=1, tauS=1,
+                                        couple='none'))
+        sim.operations.integrator = fire
+        while not(fire.converged):
+           sim.run(100)
+
+    Note:
+        The algorithm requires an integration method to update the particle
+        position and velocities. This should either be either NVE (to minimize
+        energy) or NPH (to minimize energy and relax the box). The quantity
+        minimized is in any case the energy (not the enthalpy or any other
+        quantity).
+
+    Note:
+        As a default setting, the algorithm will start with a :math:`\\delta t
+        = \\frac{1}{10} \\delta t_{max}` and attempts at least 10 search steps.
+        In practice, it was found that this prevents the simulation from making
+        too aggressive a first step, but also from quitting before having found
+        a good search direction. The minimum number of attempts can be set by
+        the user.
```

Comment:
With the defaults are documented in the function signature, you can change this to read:
```suggestion
        In practice, the default parameters prevents the simulation from making
        too aggressive a first step, but also from quitting before having found
        a good search direction. Adjust the parameters as needed for your simulations.
```